server/record/update/recordUpdateManager.js

```diff
@@ -29,6 +29,10 @@ const createRecordUpdateThread = (user, surveyId, recordUuid, preview) => {
       userIds.forEach(userId =>
         WebSocketManager.notifyUser(userId, msg.type, R.prop('content', msg))
       )
+
+      if (preview) {
```

Comment:
It's better to invoke record touch only once in recordUpdateThread (the callback is invoked 3 time)